### PR TITLE
rename second TrySend overload to TrySendAsync

### PR DESCRIPTION
**Pull request by condron**
the goal here to make the blocking and non blocking options on the ICommand Publisher interface more explicit.

### CODE REVIEW

File: src/ReactiveDomain.Messaging/Bus/Dispatcher.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public class Dispatcher : IDispatcher, IDisposable
         /// <param name="responseTimeout"></param>
         /// <param name="ackTimeout"></param>
         /// <returns>Command enqueued</returns>
-        public bool TrySend(
+        public bool TrySendAsync(
                         Command command,
                         TimeSpan? responseTimeout = null,
                         TimeSpan? ackTimeout = null)
```
Comment (dvoaviarisonld) [inline src/ReactiveDomain.Messaging/Bus/Dispatcher.cs]: You probably meant to call _queuedPublisher.TrySendAsync here?

File: src/ReactiveDomain.Messaging/Bus/ICommandPublisher.cs [hunk 2]
```diff
@@ -4,8 +4,34 @@ namespace ReactiveDomain.Messaging.Bus
 {
     public interface ICommandPublisher
     {
+        /// <summary>
+        /// Send blocks the calling thread until a command response or timeout is received
+        /// </summary>
+        /// <param name="command">The command to send</param>
+        /// <param name="exceptionMsg">The text of Exception wrapping the thrown exception, 
+        ///                 useful for displaying error information in UI applications</param>
+        /// <param name="responseTimeout">How long to wait fro completion before throwing a timeout exception and sending a cancel</param>
```
Comment (mtwoodbury) [inline src/ReactiveDomain.Messaging/Bus/ICommandPublisher.cs]: wait for completion

File: src/ReactiveDomain.Messaging/Bus/Dispatcher.cs (hunk 1 above)
Comment (condron) [reply src/ReactiveDomain.Messaging/Bus/Dispatcher.cs]: fixed

File: src/ReactiveDomain.Messaging/Bus/ICommandPublisher.cs (hunk 2 above)
Comment (condron) [reply src/ReactiveDomain.Messaging/Bus/ICommandPublisher.cs]: fixed

**Comment by condron** [comment]
updated the xml comments to include @joshkempner 's feedback and overall improve them.
also corrected the TrySendAsync call wire-up

**Pull request merged**